packages/docs/docs/billing/insurance-eligibility-checks.md

```diff
@@ -0,0 +1,135 @@
+import ExampleCode from '!!raw-loader!@site/..//examples/src/billing/insurance-eligibility-checks.ts';
+import MedplumCodeBlock from '@site/src/components/MedplumCodeBlock';
+
+# Insurance Eligibility Checks
+
+Insurance eligibility checks determine whether a patient's insurance is active, in-network, and has applicable benefits. They ensure that the provider will ultimately get compensated by the patient's insurer for a specific product or service.
+
```

Comment:
This guide glosses over the most common eligibility check which will be is this policy active and does it cover `serviceType=30' (Plan Coverage and General Benefits
Plan coverage and general benefits for the member's policy or contract.) - this particular workflow should be discussed and maybe be made into an example.
https://developers.changehealthcare.com/eligibilityandclaims/docs/use-bare-minimum-eligibility-requests

Reply:
I just pushed with updates that include this as a use case and provide examples. I also changed the use case section based on your suggestions.